### ISSUE TITLE
[HOLD react-native-webview upgrade] Leaving the thread from User B shows error message for a brief moment

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#19355]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to staging dot on web chrome and login with User A
2. Create a new workspace and invite User B
3. Now from User A account, go to announce of the workspace and send any message
4. Login with User B on any platform
5. Go to the same announce chat and reply in the thread of the message that User A has sent
6. From User B ,Now click on the workspace and click on leave room and notice that after clicking on leave room , for a very few seconds, an error message was displayed in the right hand side

## Expected Result:
Leaving the thread from User B should not show error message on the right hand side for a small moment
## Actual Result:
Leaving the thread from User B shows error message on the right hand side for a small moment

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.16-5
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/f546e7d1-af24-400c-9971-0b51d99259b4

https://github.com/Expensify/App/assets/43996225/ae4c2a8c-e083-45f9-a09b-22c395dedfe1

**Expensify/Expensify Issue URL:**
**Issue reported by:** @priya-zha
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1684382585584739

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f02534cd7b66ac3a</li>
        <li>Upwork Job ID: 1662209431573139456</li>
        <li>Last Price Increase: 2023-05-26</li>
    </ul>
</details>

**Comment by hungvu193** [comment]
Same root cause with this one: https://github.com/Expensify/App/issues/18076#issuecomment-1553998500
it's being on hold due to react-native-webview upgrade.

**Comment by mallenexpensify** [comment]
Thanks @hungvu193 do you think the plan for that is the best for here too?
>Ok so maybe a good approach would be to put this on hold until:
>
>- the rn-web version upgarde
>- the class > functional refactor
>
>And then we can re-assess the best solution after that. What do y'all think?

**Comment by hungvu193** [comment]
> Thanks @hungvu193 do you think the plan for that is the best for here too?
> 
> >Ok so maybe a good approach would be to put this on hold until:
> 
> >
> 
> >- the rn-web version upgarde
> 
> >- the class > functional refactor
> 
> >
> 
> >And then we can re-assess the best solution after that. What do y'all think?

Yes, I believe so :

**Comment by mallenexpensify** [comment]
Updated title to 
- [HOLD react-native-webview upgrade] Leaving the thread from User B shows error message for a brief moment #19355

@sobitneupane can I get a second set of eyes on the above?  Thx.

**Comment by sobitneupane** [comment]
@mallenexpensify Yup, They are similar issues. We decided to hold it until:
- the rn-web version upgarde ::: Reason: https://github.com/Expensify/App/issues/18076#issuecomment-1552859792
- the class > functional refactor ::: Reason: https://github.com/Expensify/App/issues/18076#issuecomment-1551989918

**Comment by mallenexpensify** [comment]
on hold

**Comment by mallenexpensify** [comment]
on hold

**Comment by mallenexpensify** [comment]
Still on hold but this is promising 
- https://github.com/Expensify/App/issues/16078#issuecomment-1601685060 

> Looks like there's a lot of active work happening in the RN PR.

**Comment by mallenexpensify** [comment]
Still on hold, no update on #16078

**Comment by mallenexpensify** [comment]
on hold

**Comment by mallenexpensify** [comment]
Been a bit since the below was updated
- https://github.com/Expensify/App/issues/16078

**Comment by mallenexpensify** [comment]
HOLD react-native-webview upgrade still

**Comment by mallenexpensify** [comment]
Still held

**Comment by mallenexpensify** [comment]
held

**Comment by mallenexpensify** [comment]
Still held

**Comment by mallenexpensify** [comment]
Might be waiting on this
- https://github.com/Expensify/App/issues/16078

**Comment by hungvu193** [comment]
@mallenexpensify I think this can be closed, we don't have `Leave thread` option anymore.

**Comment by mallenexpensify** [comment]
Sounds good, thanks @hungvu193
